templates/policy/route-map/node.tag/rule/node.tag/set/extcommunity-rt/node.def

```diff
@@ -2,7 +2,8 @@ type: txt
 help: Set route target value
 val_help: ASN:nn_or_IP_address:nn VPN extended community
```

Comment:
It will be more clear, if it possible.
```
<text>          ASN:NN
<x.x.x.x>:nn    IP_adress:nn
```

Reply:
Makes total sense to me. Adjusted to the same help strings as in VyOS 1.4